<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortCtl</class>
 <widget class="QWidget" name="SerialPortCtl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPortCtl</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalRegister1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName1">
         <property name="text">
          <string>COM8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect1">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalRegister2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName2">
         <property name="text">
          <string>COM9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect2">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalEncoder1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName3">
         <property name="text">
          <string>COM10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect3">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalEncoder2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName4">
         <property name="text">
          <string>COM11</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect4">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalEncoder3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName5">
         <property name="text">
          <string>COM12</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect5">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalEncoder4:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName6">
         <property name="text">
          <string>COM13</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect6">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalEncoder5:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName7">
         <property name="text">
          <string>COM14</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect7">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TerminalEncoder6:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_PortName8">
         <property name="text">
          <string>COM15</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Connect8">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="Te_Msg"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
